Pacman/Ghost.pde

```diff
@@ -0,0 +1,51 @@
+final String itemTypeGhost = "Ghost";
+String ghostImagePath = "data/ghost.jpg";
+PImage ghostImage;
+import java.util.Random;
+
+// Ghost class
+public class Ghost extends MovableItem {
+    private Random random = new Random();
+    private int ghostLife;
+    
+    // Constructor
+    public Ghost(String name, float x, float y) {
+        super(name, x, y);
+        ghostLife = 3;
+        setSpeed(30.0); // set Ghost speed
+        randomizeDirection(); // Start by a random direction
+    }
+
+    @Override
+    public void evolve(GameInfo gInfo) {
+        //  Randomly change direction
+        if (random.nextInt(35) == 0) {
+            // probability of changing direction on each call
+            randomizeDirection();
+        }
+        
+        // keep moving
+        move(gInfo);
```

Comment:
you can just call `super.evolve(gInfo)` here. then no need to override the `move` method.